yolog/main.py

```diff
@@ -45,5 +46,7 @@ def main():
     git_arguments = sys.argv[1:]
     if git_arguments and (sys.argv[1] == "-h" or sys.argv[1] == "--help"):
         print(help_description)
+    elif git_arguments and (sys.argv[1] == "-config"):
+    	ConfigHandler()
```

Comment:
Also, we are doing:
```
config_handler = ConfigHandler("~/.yolog/config.ini")
```

so remove the `path` variable from `yolog_updater.py` and pass it as argument in `__init__` method itself